### ISSUE TITLE
Privacy Manifest is not recognized.

### ISSUE THREAD

**Issue by wlxo0401** [amplitude/Amplitude-iOS#482]
Occurs while developing iOS apps

## Expected Behavior
When I checked 'PrivacyReport' after proceeding Arcive, there was no Amplitude. 

## Current Behavior
As far as I know, the current content of 'Amplitude PrivacyInfo' should be marked on 'PrivacyReport'.

## Possible Solution
SPM package needs to be modified and podspec needs to be modified as well. 

I don't know if this is the right answer, but all the other libraries have gone through this process.

## Steps to Reproduce
1. Install Amplitude 8.18.0 (I used SPM)
2. Verify the presence of PrivacyInfo files in Amplitude
3. Proceed with the App Project Archive
4. Output a Privacy Report from the Organizer screen
5. Check PDF

## Environment
- SDK Version: 8.18.0
- Device: Mac(Build)
- OS Version: MacOs 14.2.1

**Comment by Mercy811** [comment]
Hi @wlxo0401, thanks for choosing Amplitude. How did you install Amplitude SDK? You have to embed it to have the privacy manifest loaded to your archive to generate a privacy report. Read more [here](https://forums.developer.apple.com/forums/thread/734971).

You can also copy the fields from Amplitude's [privacy manifest](https://github.com/amplitude/Amplitude-iOS/blob/main/Sources/PrivacyInfo.xcprivacy) to your project's `Sources/PrivacyInfo.xcprivacy`.

**Comment by wlxo0401** [comment]
> Hi @wlxo0401, thanks for choosing Amplitude. How did you install Amplitude SDK? You have to embed it to have the privacy manifest loaded to your archive to generate a privacy report. Read more [here](https://forums.developer.apple.com/forums/thread/734971).
> 
> You can also copy the fields from Amplitude's [privacy manifest](https://github.com/amplitude/Amplitude-iOS/blob/main/Sources/PrivacyInfo.xcprivacy) to your project's `Sources/PrivacyInfo.xcprivacy`.

@Mercy811 

I install the library through SPM.

If I look at other libraries, when you add PrivacyInfo Modify "Package" and "podspec" together.
[podspec](https://github.com/onevcat/Kingfisher/commit/c0fcbc979eeac7f9067601ec0403f580a7a6e3a8), [package](https://github.com/onevcat/Kingfisher/commit/a4bd0640556d4137859ae329ac6ddfc7ec6478ce)

But you mean to copy 'PrivacyManifest' content of 'Amplitude' to 'PrivacyInfo' of my app project??

**Comment by Mercy811** [comment]
Hi @wlxo0401, I will take a closer look into it and keep you updated. Thanks.

**Comment by wlxo0401** [comment]
> Hi @wlxo0401, I will take a closer look into it and keep you updated. Thanks.

Yes, thank you. And one more thing I'm curious about 

From what I see, when you add 'PrivacyInfo' in 'Amplitude', it seems that you added it to 'Package' right away. Is this right?? 

In my search, other libraries seem to have added 'PrivacyInfo' to 'xcworksapce' or 'xcodeproj'.

The reason why I'm guessing like this is that I checked the commitment at the time of adding 'PrivacyInfo' to each library. 

And I think they set 'PrivacyInfo' target as well. (Are they targeting the library???)

I don't know what difference this makes. Maybe it doesn't make a difference. Or maybe the Amplitude way is right. 

I'm just telling you the difference from the other libraries I found when you added 'PrivacyInfo'... 

Looking it up, Apple doesn't seem to give clear guidelines. 

I also didn't check all the libraries.
